docs/api/qiskit/0.19/_toc.json

```diff
@@ -3789,171 +3789,171 @@
       "children": [
         {
           "title": "0.19",
-          "url": "/api/qiskit/0.19/release-notes/0.19"
+          "url": "/api/qiskit/release-notes/0.19"
         },
```

Comment:
Oohh I didn't catch this until now. Note that we repeat the 0.19 entry twice in this list. This top-level one isn't necessary.

This PR is already so big that maybe it's best to fix in a follow up PR?